client/evm.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"time"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+)
+
+type NodeKeysBundle struct {
+	OCR2Key client.OCR2Key
+	PeerID  string
+	TXKey   client.TxKey
+	P2PKeys client.P2PKeys
+}
+
+type Node struct {
+	ID        int32     `json:"ID"`
+	Name      string    `json:"Name"`
+	ChainID   string    `json:"ChainID"`
+	URL       string    `json:"URL"`
+	CreatedAt time.Time `json:"CreatedAt"`
+	UpdatedAt time.Time `json:"UpdatedAt"`
+}
+
+func CreateNodeKeysBundle(nodes []*client.Chainlink, chainName string) ([]NodeKeysBundle, error) {
```

Comment:
Where was this code moved from?

It probably fits more to move it as part of the Chainlink client in the core repo `"github.com/smartcontractkit/chainlink/integration-tests/client"` (next to the code it uses)

Reply:
This was code that was adjusted based on how keys were created in Solana, we had no methods for EVM based chains so idea was to move it here since the setup will always be same for evm chains